### DISCUSSION TITLE
Templates Parsing not working

### DISCUSSION THREAD

**Discussion by ih8snow** [BryanJenksCommunity/FAQ#60] [Q&A] [answered]
### Thanks###
Hi Bryan, 
thanks for all the great videos!
I'm trying to catch up with áll the things your are showing and coyping what I need for my workflow.
Hint: Would love to see some "How should should / could start videos = best practices for newbies :-)

Still a new user and at the beginning of this journey. I thought it's a good idea to use templates for a good structure, so I started with your latest version. 
But...

### Question

When I use your templates in my Obsidian (0.11.13) with templater 1.51 it's not parsing the data.
i.e. <%tp.file.title%> does not show the file name and <%tp.date.now()%>] does not show the date.

I tried to Templates: Replace templates in the active file and then I saw this note coming up:
templater error Template parsing error, aborting check concole for more informations

I have no clue what "console" is meant here.

Do you any guidance for me?

Best,
Jörn

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
Hey! when you insert the templates, are you using a hotkey or a palette command for the built in core template plugin? or tempalter. because a common error I have made is to insert templates with the core plugin when I meant to use tempalter.

console is the developer consol, on mac its cmd+opt+I to view it, not sure about windows or Linux but it's viewing the developer console that looks like this:

![image](https://user-images.githubusercontent.com/29872822/114818998-9022d800-9d71-11eb-9977-2ca571f635f4.png)

**Reply by ih8snow** [reply]
Hi Bryan,
I included the template via the ctrl-p option (=cmd-p) Templater: Insert Template

But it's all ok, error was between my ears... I crashed some of the metadata fields...should be more carefully :-)

Thanks!

Jörn

**Reply by tallguyjenks** [reply]
awesome! have fun :)
